### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by ManognaMutyala**
## Description

Updated strings as per the alert dialog box for confirming logout

## Related issues and discussion
#631

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -222,6 +222,9 @@
     <string name="error_invalid_password">The password needs to be at least 6 characters long.</string>
     <string name="error_field_required">This field is required.</string>
     <string name="action_preferences">Preferences</string>
+     <string name="ok">OK</string>
+     <string name="cancel">CANCEL</string>
+    <string name="pressed_cancel">Log out cancelled</string>
     <string name="prefs_scan_startup">Scan on startup</string>
```
Comment (samagra14) [inline app/src/main/res/values/strings.xml]: Follow proper formatting if you want your travis build to pass.

Comment (teolemon) [reply app/src/main/res/values/strings.xml]: Can you merge this pull request with the one introducing the feature ?

Comment (ManognaMutyala) [reply app/src/main/res/values/strings.xml]: Yeah sure!!

Comment (ManognaMutyala) [reply app/src/main/res/values/strings.xml]: Thanks @samagra14  for the suggestions

Comment (ManognaMutyala) [reply app/src/main/res/values/strings.xml]: Yeah sure @teolemon

**Comment by Karljoones** [comment]
@ManognaMutyala since this is going to be added to PR #744 can we go ahead with PR closure of this PR(#745)?

**Comment by ManognaMutyala** [comment]
Okay @Karljoones

**Pull request closed without merging**